INSTRUCTION.md

```diff
@@ -0,0 +1,7 @@
+kubectl get secrets -n mysql
+kubectl get configmap -n mysql
+kubectl get headless -n mysql
```

Comment:
The command `kubectl get headless` is incorrect. Kubernetes does not have a resource type called 'headless'. Use `kubectl get service` to list services, including headless services.